test/globals.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+global.chai = require('chai')
+global.should = global.chai.should()
+global.sinon = require('sinon')
+global.proxyquire = require('proxyquire')
```

Comment:
I find it useful, take them out in a subsequent PR that shows how useful it is not to globalise them if you like